EIPS/eip-7668.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7668
+title: Add Max Epoch Churn Limit
+description: Modify the churn limit function to upper bound it to a max value
+author: dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7668-add-max-epoch-churn-limit/15709
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-07
+---
+
+## Abstract
+
+Limit Beacon chain growth from exponential to linear to mitigate the risks of very high levels of total ETH supply staked.
+
+## Motivation
+
+With a full deposit queue, the share of ETH supply staked will reach 50% by May 2024, 75% by September 2024, and 100% by December 2024. While yields decrease as the validator set size increases, the APY at 100% ETH supply staked is 1.6% for consensus rewards only excluding MEV. This yield can be sufficient to attract capital such that the equilibrium point of the validator set size is close to its maximum possible.
```

Comment:
I would use the following language to make it more explicit the assumption:
> In the event that the deposit queue stays 100% full

Reply:
"1.6% due to issuance, excluding transaction fees/MEV"

Reply:
> This yield can be sufficient to attract capital such that the equilibrium point of the validator set size is close to its maximum possible.

I don't think this claim is substantiated. 1.6% yield is not necessarily sufficient to attract capital. At just 1.6%, capital can easily find way less risky yields -- e.g. bonds and t-notes.

"This small yield does not necessarily dissuade additional capital staking due to the often much higher and unpredictable yields from MEV"